notes/sorts.md

```diff
@@ -41,8 +41,8 @@ int partition(array, lo, hi)
    int i = lo
    for (int j = lo; j < hi; j++)
       if (array[j] <= pivot)
-         swap(array[i], array[j])
+         swap(array, i, j)
```

Comment:
Since this is pseudocode, your update makes this less readable. Remove this change.